### DISCUSSION TITLE
New empirical evidence for tired light: the redshift bias of galactic rotation

### DISCUSSION THREAD

**Discussion by LifeIsStrange** [a-cosmology-group/acg#254] [Astrophysics and Physics]
So I already talked some time ago about one of the most extreme mystery in the universe, that happens to be talked by no one,
The observation that there is a spin alignment bias in the rotations curves of galaxies when split between clockwise and counterclockwise.
They have the same relative frequency but Lior Shamir shows that they are aligned relative to a distant center (center of the universe? New rotational bias physics?)

https://arxiv.org/abs/2407.20487
He just published a new paper on the topic, again, finding that redshift at the same distance, change based on the rotation direction of a galaxy.
Moreover what is new to me, is that he shows that the redshift bias between clockwise and counterclockwise galaxies, INCREASE with redshift.
This is the first direct evidence of tired light even though it is mediated by physics we don't fully understand.
There are many lines of evidence for TL but very few that are as direct (without inferences and biases) as this one.
He reproduced the simple observation on multiple datasets.
Shamir also shows other major discoveries:

Clockwise ratio increase with redshift which is a brightness bias enhanced by tired light 
> That observation can also be related to the far higher num­ber of galaxies that rotate in the opposite direction relative to the Milky Way as imaged by JWST (Shamir, 2024a). The JWST deep field images show a 140% more galaxies that rotate in the opposite direction relative to the Milky Way (Shamir, 2024a). A possible explanation is that galaxies that rotate in the opposite direction relative to the Milky Way are brighter due to the Doppler shift effect. Due to TL, the brightness difference gets more significant when the galaxies are more distant

It solves the Hubble tension 
> Because H0 is determined by using the redshift, a redshift bias can also explain the observed H0 tension. For instance, when using the SH0ES catalog (Khetan et al., 2021) of Ia supernovae by just selecting the galaxies that rotate in the same direction as the Milky Way, H0 drops from ∼73.7 to ∼69.05 km/sMpc−1 (McAdam and Shamir, 2023), which is within statistical error from the H0 as observed by the CMB. When using just SH0ES galaxies that rotate in the opposite direction relative to the Milky Way, the H0 increases to ∼74.2 km/sMpc−1 (McAdam and Shamir, 2023).

Explaining how rotation affects redshift is key to major insights AND evidence for the mechanisms of new physics. 
For example, as I said previously, there are two distinct tests of gravity in lab that are made on a rotating object (one being mossbauer) and they both shows an excess of observed time dilation redshift when upon rotation at specific accelerations.
This is highly related to the quantized theory of inertia and possibly to rindler horizons and the casimir effect. 
I conjecture that it is the same force as observer on mossbauer that explains this cosmological redshift.
Moreover, a related gravity experiment is the Sagnac effect. 
And my mind is completely blown because there is a groundbreaking paper that shows that the Sagnac effect explains rotational redshift and that it is the dominant redshift mechanism in the universe. This would make redshift not a reliable indicator of distance (to what extent?) and cosmic time dilation, an illusion.
https://www.medjchem.com/index.php/Medjphysics/article/download/518/556
Related paper:
https://ej-physics.org/index.php/ejphysics/article/view/308
https://arxiv.org/abs/2403.17151v1

Additional snippets from Shamir's paper:
> Although alignment in galaxy spin directions is expected (d’Assignies D et al., 2022; Kraljic et al., 2020), it does not expect to form a cosmological-scale axis. If such axis in­deed exists and it is not driven by the impact of the ro­tational velocity on the redshift measurements, it can be linked with theories such as dipole cosmology or rotating universe. Theories that assume a universe rotat­ing around a cosmological-scale axis include Black Hole Cosmology and ellipsoidal universe

Off topic but dust polarization has been published for the spider cmb balloon
https://arxiv.org/abs/2407.20982
It is very interesting because they find dust roughly inconsistent with a modified black body, which strengthen the case for the CMB being generated by dust.

https://arxiv.org/abs/0805.2543
https://www.semanticscholar.org/paper/Observation-of-the-Zero-Doppler-Effect-Ran-Zhang/ce0253bfad1c0221b2259caa7ac71f26843c87e3
https://www.semanticscholar.org/paper/Research-on-Detecting-Targets%E2%80%99-Accelerations-Based-Guo-Lei/8f4b9be35066ccf9f350d03ac1fa8383bc3825ad
https://www.semanticscholar.org/paper/Gravitational-Redshift-and-Deflection-of-Slow-Light-Dressel-Rajeev/2824b837c39bd882c029e7d85e772deb691717c5#citing-papers
https://www.researchgate.net/publication/299354738_Rotational_Doppler_effect_in_nonlinear_optics
https://www.researchgate.net/publication/357784834_Inner_Workings_Astronomers_uncover_new_way_to_measure_the_speed_of_stars
https://www.researchgate.net/publication/374229405_Cosmological_Redshift_via_Non-Linear_Frequency_Down-Conversion_of_Electromagnetic_Radiation
https://arxiv.org/pdf/2203.02282
https://www.researchgate.net/publication/1966620_The_Relativistic_Sagnac_Effect_Two_Derivations
https://www.researchgate.net/publication/378379267_Gravitational_effects_in_a_superconducting_film_struck_by_a_laser_pulse?_tp=eyJjb250ZXh0Ijp7ImZpcnN0UGFnZSI6InB1YmxpY2F0aW9uIiwicGFnZSI6InB1YmxpY2F0aW9uIn19
https://hal.science/hal-00834705
https://theses.hal.science/tel-01508504v1/html_references
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10490657/

**Comment by ExpEarth** [comment]
This is an interesting paper for sure, but it's also quite confusing.  I can see why a galaxy rotating in the _opposite_ direction as the Milky Way might appear more redshifted than one rotating in the same sense, as there would be an extra relativistic term.  But why would a galaxy rotating in the _same_ direction as the Milky Way be more redshifted?  The increased redshift differential with higher redshift could be explained as a composite redshift effect.  I'm not connecting all the dots here and I'm not sure the author has either (despite the huge amount of work he put in!)

**Reply by RedshiftDrift** [reply]
The argument used in the paper seems to be along these lines:
- light emitted by rotating objects can show a redshift (although very small and for a special polarization produced in the lab)
- the redshift of galaxies could be produced by their rotation
- the effect is small, but we don't understand galaxy rotation very well so the effect could be larger than we think
- we don't understand tired-light redshift very well either, so this might be the cause of galaxy redshifts.

This is a complete fallacy, some sort of argument from ignorance combined with the false dilemma fallacy.
That paper does not show anything.

@ExpEarth, I agree the author is not connecting the dots...

**Reply by Pierre314159** [reply]
Indeed, the logic is rather flaky...

There are, however, some very conspicuous truths that should not be dismissed:

- we don't understand galaxy rotation (very well)
- we don't understand tired light redshift (very well)

And an other one that I fished out of the article:

- tired light redshift is not an indication of distance

**Comment by LifeIsStrange** [comment]
Yet another paper from shamir on anomalous galaxy rotation, this time from JWST data
https://academic.oup.com/mnras/article/538/1/76/8019798?login=false

**Comment by ExpEarth** [comment]
Here is a news story about it.  Shamir thinks the observations could support the idea that we are living inside a black hole universe.  The general idea is that if the universe arose as a rotating black hole, there could have been a preferential rotation seen in galaxies and also an axis of rotation.  Somehow it all has to connect up with the motions of cosmic filaments and I don't think he has made the connections yet.  It's pretty amazing that he did this single-handedly using the JWST data.

https://www.space.com/space-exploration/james-webb-space-telescope/is-our-universe-trapped-inside-a-black-hole-this-james-webb-space-telescope-discovery-might-blow-your-mind

**Reply by ExpEarth** [reply]
Now Anton has done his own video on Shamir's rotating galaxies.  He makes some interesting points.  Apparently, there were studies 10 years ago showing that quasars also have a preferred rotation pattern.  Anton links the quasar rotations to the rotation of the giant cosmic filaments which contain them.  Thus, he argues that the preferred rotations in galaxies and quasars merely reflect the rotations of the filaments.

However, he does not go deeper and ask what is causing the filaments to rotate.  If you think of a rotating shell black hole made up initially of plasma, then the rotating bands of plasma could have formed into tight belts (as in Lerner's model).  Their rotation would have a preferred orientation relative to the whole universe.  The question then becomes how did these filaments migrate towards the interior and why?

 https://www.youtube.com/watch?v=dmdrfKETKAE

**Reply by RedshiftDrift** [reply]
Do you mean Peratt's model? A. L. Peratt, “Evolution of the Plasma Universe: II. The Formation of Systems of Galaxies,” IEEE Trans. Plasma Sci., vol. 14, no. 6, pp. 763–778, 1986, doi: [10.1109/TPS.1986.4316625](https://doi.org/10.1109/TPS.1986.4316625).

**Reply by ExpEarth** [reply]
I was meaning more generally, on the cosmic level.  Suppose that the observable universe is a shell black hole universe and that it originated with most of the mass in the shell region.  As a black hole we would expect some rotation.  Now if a good portion of the gas is a plasma, then this rotation would cause the plasma to pinch up into rotating belts of plasma, as in Alfven's, Peratt's and Lerner's models.  These rotating belts could be the original cosmic filaments.  If they were to move into the centre region, for whatever reason, then the angular momentum that the rotating belt initially had might be channeled into rotation of filaments along the axis and in galaxies attached to the filaments.  Just tossing this out there.

**Comment by LifeIsStrange** [comment]
Interesting review of systematics about redshift independent measures of distance which as usual are the core unknowns in cosmology that affect everything. In other words the less talked about analogues of the Hubble tension.
https://arxiv.org/abs/2503.13658

**Comment by LifeIsStrange** [comment]
Interesting rebuttal of the findings of Lior Shamir, especially the first link
I wonder who is right
https://www.reddit.com/r/cosmology/comments/1ja9i53/comment/mhk61bo/?utm_source=share&utm_medium=mweb3x&utm_name=mweb3xcss&utm_term=1&utm_content=share_button

**Reply by RedshiftDrift** [reply]
I am collaborating with Shamir to describe this effect from an optics point of view.  Since *all galaxies have a toroidal field rotating in the counterclockwise direction* (as seen from above the disk, see image below), this could polarize light (via Zeeman effect in the interstellar medium) and produce a rotation-dependent attenuation of light in the magnetic field of the Milky Way.  This could be why Shamir's rotation anisotropy is aligned with the Milky Way axis.  The dependence with redshift claimed by Shamir would only be the result of a Malmquist bias that gets worse with fainter galaxies at large redshifts.

[Patel and Desmond](https://doi.org/10.1093/mnras/stae2158) use data sets at smaller redshifts, where a Malmquist bias is less likely to produce the effect.  I'll ask Lior what he thinks about the Patel and Desmond analysis.

I think the rotation asymmetry may be an optical effect.  However, the "rotating universe" hypothesis is nonsense and would only add another unlikely coincidence in cosmology where the Milky Way rotation would be anti-parallel to that of the universe.

![Toroidal-Bo](https://github.com/user-attachments/assets/48ee4a3f-6283-41ca-ae9c-ca9f9d97ab75)

**Reply by RedshiftDrift** [reply]
Lior Shamir responded very quickly, he already has a comment showing the ineptitude of the [Patel and Desmond](https://doi.org/10.1093/mnras/stae2158) paper!

Here is part of Lior's response:
> I am aware of that paper. I addressed it in Section 4 of https://doi.org/10.1093/mnras/staf292  (the last 3 paragraphs of Section 4).
> There is a more detailed version of that explanation in a comment: https://arxiv.org/abs/2404.13864

Lior also tried to reproduce the results of Patel and Desmond, without success.  This is explained here https://people.cs.ksu.edu/~lshamir/data/patel_desmond/.  Same conclusion for Iye et al. (2021) explained in https://people.cs.ksu.edu/~lshamir/data/iye_et_al/

The author of the [reddit comment](https://www.reddit.com/r/cosmology/comments/1ja9i53/comment/mhk61bo/?utm_source=share&utm_medium=mweb3x&utm_name=mweb3xcss&utm_term=1&utm_content=share_button) did not understand Shamir's 2024 paper.  Both the 2024 and 2025 papers agree on the direction of rotation for the more numerous galaxies.

The reddit post belittles Shamir by writing *"solo author (a computer scientist) has made many similar claims...  Some of these claims have been followed up by astronomers"*, without realizing that astronomers use the analysis software written by Shamir!

Another example of the confirmation bias of mainstream astronomers who don't want to admit they were fooled by optical illusions.

**Comment by HanDeBruijn** [comment]
@LifeIsStrange [writes](https://github.com/orgs/a-cosmology-group/discussions/254#discussion-6998055):
>The observation that there is a spin alignment bias in the rotations curves of galaxies when split between clockwise and counterclockwise.

Take a piece of transparent paper. Draw a circle on it, with an arrow indicating the rotation direction. Let's say the spin  is clockwise. Now look through the transparent from the other side. What you will see then is that the spin has become counterclockwise. Which of the two rotations is the "true" one?

@RedshiftDrift [writes](https://github.com/orgs/a-cosmology-group/discussions/254#discussioncomment-12924696) (bold by me):
>Since _all galaxies have a toroidal field rotating in the counterclockwise direction_ (as seen **from above** the disk [ .. ] )

Is there a difference between "from above" and "from below" with a galaxy as seen from the earth?
Related: [Cross product](https://en.wikipedia.org/wiki/Cross_product) (Wikipedia).

**Reply by RedshiftDrift** [reply]
@HanDeBruijn 
By "from above", I mean looking at the magnetic field on our side of the observed galaxy.  The B-field will always be in the counterclockwise direction (as shown on the image [here](https://github.com/orgs/a-cosmology-group/discussions/254#discussioncomment-12924696)).  It is related to the 'cross product' because the B-field is produced by a current via the [Biot-Savart law](https://en.wikipedia.org/wiki/Biot%E2%80%93Savart_law#Electric_currents_(along_a_closed_curve/wire)).

There is no difference between "from above" and "from below" in terms of the electron current along $\hat z$ and $\hat r$ [cylindrical coordinates] and the magnetic field it generates.  Electrons moving ***toward the galaxy*** along the rotation axis, and moving away from the galaxy on the plane of the disk, produce the anti-toroidal field as observed for the Milky Way.   (Not a 'jet' of electrons, [a 'sink' as I have argued](https://github.com/orgs/a-cosmology-group/discussions/144#discussioncomment-7744291) with @sahil5d.  A jet of electrons would produce a clockwise toroidal magnetic field that is not observed!)

However, the smaller, dipolar component of the magnetic field $B_z$ around a galaxy defines a direction that changes if the galaxy is flipped about its rotation axis $\hat z \rightarrow -\hat z$.

**Reply by Francois-Zinserling** [reply]
@RedshiftDrift I'm asking a hypothetic question: If the galaxy were to eject a jet of electrons 'up' and positrons 'down' (or vice-versa) there would be no toroid but just a magnetic field as if there is a current in a wire, right?

What I am not sure about is in the image the magnetic lines (blue) appear like the field lines of a normal N-S magnetic field. Lines would be continuous (curved of course) from N to S through the disc as is shown.
If the blue lines are from a N-S magnetic field, then there shouldn't be a toroid, but if there definitely is a toroid then the blue lines cannot meet at the disc. What am I missing?

Is the toroid a confirmed observation? Also for other galaxies?

**Reply by HanDeBruijn** [reply]
If visuals (in words) are not convincing, let's try it the mathematical way. Let $\vec{a}$ and $\vec{b}$ be two vectors in the XY plane with an angle $\theta$ between them. And let us write for the z-component of the cross product, with help of the unit vector $\vec{k}$ :

$$
\vec{k} \cdot \left( \vec{a} \times \vec{b} \right) = |\vec{a}||\vec{b}|\sin(\theta) \quad \mbox{with} \quad \theta \gt 0
$$

Physically this corresponds with an observer, heads up in positive $z$ direction, looking at a galaxy rotating over an angle $\theta$, counter clockwise. Now let's turn the observer upside down. It may be sufficient for the purpose to let him or her travel from Canada to some country in the Southern hemisphere. Then the z-component of the cross product becomes pointed into negative direction, giving

$$
-\vec{k} \cdot \left( \vec{a} \times \vec{b} \right) = - |\vec{a}||\vec{b}|\sin(\theta) = |\vec{a}||\vec{b}|\sin(-\theta)
$$

With other words, our observer now sees the same galaxy rotating in the opposite direction $(-\theta)$ , that is clockwise.

I don't understand what scientific value can be attached to observations that are **entirely dependent on orientation of the observer**.  @RedshiftDrift . I may be missing some other things, but it also escapes me how your reply is related to the main subject of this thread.

**Update.** Define a vector $\vec{r}$, uniformly rotating in the XY plane, by

$$
\vec{r} = \begin{bmatrix}
\cos(\theta) \\
\sin(\theta)
\end{bmatrix}
$$

Take time derivative, giving angular velocity $\dot{\theta}$ :

$$
\dot{\vec{r}} = \begin{bmatrix}
-\sin(\theta).\dot{\theta} \\
\cos(\theta).\dot{\theta}
\end{bmatrix} = \begin{bmatrix}
0 & -\dot{\theta} \\
+\dot{\theta} & 0
\end{bmatrix} \begin{bmatrix}
\cos(\theta) \\
\sin(\theta)
\end{bmatrix}
$$

Differentiate again:

$$
\ddot{\vec{r}} = \begin{bmatrix}
0 & -\ddot{\theta} \\
+\ddot{\theta} & 0
\end{bmatrix} \begin{bmatrix}
\cos(\theta) \\
\sin(\theta)
\end{bmatrix}
+\left(\begin{bmatrix}
0 & -\dot{\theta} \\
+\dot{\theta} & 0
\end{bmatrix}\right)^2
\begin{bmatrix}
\cos(\theta) \\
\sin(\theta)
\end{bmatrix}
$$

Suppose that rotation is uniform, ipse est $\ddot{\theta} = 0$ . With $\omega = \dot{\theta}$ the remaining term can be written as

$$
\ddot{\vec{r}} = \begin{bmatrix}
0 & -\omega \\
+\omega & 0
\end{bmatrix} \begin{bmatrix}
0 & -\omega \\
+\omega & 0
\end{bmatrix} \begin{bmatrix}
\cos(\theta) \\
\sin(\theta)
\end{bmatrix} =
\begin{bmatrix}
-\omega^2 & 0 \\
0 & -\\omega^2
\end{bmatrix}
\begin{bmatrix}
\cos(\theta) \\
\sin(\theta)
\end{bmatrix} 
$$

$$
\Longrightarrow \quad \ddot{\vec{r}} = -\omega^2 \vec{r}
$$

which is a well known result for the acceleration in uniform circular motion. Now pay special attention to the square in $\omega^2$. As a consequence, **not a single force in nature can distinguish between clockwise or counter clockwise rotation** if angular velocities are uniform and equal in magnitude.

**Reply by HanDeBruijn** [reply]
A picture says more than a thousand words:

![ogen](https://github.com/user-attachments/assets/db826abe-5e36-466c-b221-9f829500bb8b)

**Reply by RedshiftDrift** [reply]
@Francois-Zinserling 
> I'm asking a hypothetic question: If the galaxy were to eject a jet of electrons 'up' and positrons 'down' (or vice-versa) there would be no toroid but just a magnetic field as if there is a current in a wire, right?

I assume you mean "electrons moving up along $+z$ on the $z>0$ side of the galaxy, and positrons moving down along $-z$ on the $z<0$ side of the galaxy."
Like this:
![wire-field](https://github.com/user-attachments/assets/f06c098b-09ac-4e2c-8bf0-56c495aae1c1)

  For that hypothesis, then both currents are in the same direction $-\hat z$ and the magnetic field is as you say, "as if there is a current going towaed $-\hat z$ in a wire", that wire being along the $z$ axis.  (That field is not a pair of counter-rotating toroidal fields.)

$\ $
> What I am not sure about is in the image the magnetic lines (blue) appear like the field lines of a normal N-S magnetic field. Lines would be continuous (curved of course) from N to S through the disc as is shown.

The blue lines are similar to those produced by a N-S magnet - a *dipolar magnetic field*.  A current rotating along $\hat\phi$ (cylindrical coordinates) in the plane of the galaxy (in green on image below) also produces a dipolar field, and the direction of the field (blue lines) depends on the rotation direction of the current.

> If the blue lines are from a N-S magnetic field, then there shouldn't be a toroid, but if there definitely is a toroid then the blue lines cannot meet at the disc. What am I missing?

What's causing the counter-rotating toroidal fields (red) are electrons moving along $z$ **toward** the centre of the galaxy (a **drain** of electrons above and below the galaxy shown in orange below) and then flowing along the radial direction $+\hat r$ in the plane of the galaxy.

I think what you'er missing is that all these currents exist at the same time, and the total magnetic field is the sum of the counter-rotating toroidal fields (red) *plus* the dipolar field (blue).

$\ $
> Is the toroid a confirmed observation? Also for other galaxies?

Yes for the Milky Way, I took the image from this paper https://dx.doi.org/10.3847/1538-4357/ad3a61, Figure 8.
I don't know if this is confirmed experimentally for other galaxies, I haven't had the time to do a good litterature search.

However, the prediction I make using "stimulated transfers" is that the "drain/centrifuge" motion of the electrons is the same for all galaxies, and the toroidal components of the field will always be in the same direction.

![Toroidal-Bo-currents](https://github.com/user-attachments/assets/9b3c9596-5134-4238-b596-551e920b8183)

**Reply by RedshiftDrift** [reply]
@HanDeBruijn 
> I may be missing some other things...

In the Shamir paper, the rotation of distant galaxies is not measured with respect to the observer, but instead compared to the direction of rotation of the Milky Way.

$\ $
>  not a single force in nature can distinguish between clockwise or counter clockwise rotation

The mathematical way is not convincing if it ignores some physical laws, as a result your statement is wrong.  The [Lorentz force](https://en.wikipedia.org/wiki/Lorentz_force) and the [Zeeman effect on light polarization](https://en.wikipedia.org/wiki/Zeeman_effect) are both sensitive to rotation direction.  That's how the direction of the magnetic fields around the Milky Way was measured.

$\ $
> I don't understand what scientific value can be attached to observations that are entirely dependent on orientation of the observer.
> it also escapes me how your reply is related to the main subject of this thread 

By going from Ottawa to Adelaide, I would observe the inverted image below.  My point is that while the dipolar field (blue) is inverted, *the toroidal field (red) is the same as in the* [image above](https://github.com/orgs/a-cosmology-group/discussions/254#discussioncomment-12956619) and therefore **independent** *of the orientation of the observer.*

Because some observations are independent of the orientation of the observer, the scientific value of this is that it unambiguously tells us the direction of motion of electrons (orange) along the axis of rotation of a galaxy: *it's a drain!*

![Toroidal-Bo-currents-mirror](https://github.com/user-attachments/assets/6dd5f453-1e43-4cae-b9b9-d52508a93b91)

**Reply by HanDeBruijn** [reply]
## Front : clockwise
![9s3bkd](https://github.com/user-attachments/assets/e84b8ace-f8ca-4069-970a-0f06de5c0895)
## Back : counter clockwise
![9s3f4m](https://github.com/user-attachments/assets/361e924e-a1a9-479b-8b37-ffe4c8e04480)

**Reply by HanDeBruijn** [reply]
@RedshiftDrift writes:
>compared to the direction of rotation of the Milky Way.

Which are the **green** arrowed lines in your pictures. When viewed "upside down" the Milky Way rotates the other way around. Unfortunately though, _there are no upside downs in cosmos_. And it still escapes me how the **red** arrowed lines are related to the green ones.

>your statement is wrong.

Nowhere in my replies there is a reference to electricity and magnetism; I do not understand what you are talking about.

**Reply by budrap00** [reply]
My problem with all of this is the toroidal field which is an effect found in a very specific laboratory arrangement:
https://www.sciencefacts.net/toroid-magnetic-field.html

A toroid is a physical structure with a conducting wire wrapping that creates the confined toroidal field when a current is present in the wire. I suppose there may be analogous systems extant in the wild beyond the laboratory but I don't know of any examples. A disk galaxy is obviously not a toroid so I don't see how a confined toroidal field arises absent a toroidal defining wrapping current of some sort.

**Reply by RedshiftDrift** [reply]
@HanDeBruijn

When the 'mechanical' rotation direction of a galaxy is compared to the rotation of the Milky Way, the orientation or position of the observer does not matter.
Example below: galaxies rotate ***in the 'opposite' direction of the Milky Way***, as seen from whichever point of view.
That's what Shamir reports in his paper.
![rotation-same](https://github.com/user-attachments/assets/73ded601-326c-4528-9f48-f877bcb42c82)

The rotation of a galaxy is analyzed via light, or "electricity and magnetism", that's how we know galaxies exist.  When you talk about rotation of galaxies, it is therefore implied that electricity and magnetism are involved in the observation.  Your replies have an implicit dependence on electricity and magnetism because we are talking about real, observed galaxies, not hypothetical mathematical constructs.
Science must include all aspects of observations, even the optical illusions that make things look different.

@budrap00 a "toroidal" field can be generated with currents that are different from the example you give.  It's a qualitative description that I make here, I am not talking about an exact mathematical toroid.  Galaxies no not have a toroidal shape.  However, a current as drawn in [orange in the image](https://github.com/orgs/a-cosmology-group/discussions/254#discussioncomment-12956804) will generate a *pair* of toroidal-looking fields that circulate in opposite directions.  The current does not have to wrap the toroidal shape exactly as shown.
For galaxies, the electrons flow far away from the galaxy before returning to the 'poles' along large loops.  It is also possible (and very likely) that multiple galaxies are connected in this electrical circuit, but since the generated magnetic field is significantly large near the most intense part of the currents, the "toroids" will appear near the galaxies.  The shape of the magnetic field on one side of a galaxy is probably not even as perfect as this doughnut
![doughnut](https://github.com/user-attachments/assets/b1ac4c91-449e-48bd-91bf-ea166e5caded)
The red lines in the images are meant to represent the position of the toroid-like shape.  The most important property is the direction of rotation of the field.

@HanDeBruijn The red lines are produced by the current shown in orange.  The red magnetic field looks the same for any observer.
Here is a cleaned up version showing only the counter-rotating toroidal fields.
The toroidal components of the magnetic field look the same for all galaxies, from any point of view, e.g.:
![anti-toroid](https://github.com/user-attachments/assets/5b1e6a58-91f2-4950-850b-193967e2c2cd)
That's a prediction I make, and the measured magnetic field around the Milky Way conforms to this prediction (I am aware there is a 50% chance of agreement on a single measurement...)
While there is no "up" and "down" in space, a rotation of an image by 180 degrees shows the 'upside down' version of that image.

For an observer looking at a galaxy face-on, the toroidal magnetic field always appears to circulate in the counter-clockwise direction.
![toroidal](https://github.com/user-attachments/assets/1fe64c1f-6053-4256-8fab-afbf02d736ac)
The toroidal component of the magnetic field that is on the opposite side of the galaxy cannot be observed because no light going through it reaches us.
But contrary to the images of your clock, looking at the back of the galaxy would show the same thing because we would see the anti-toroidal field!

Galaxy rotation produces the dipolar field (in blue in the [images above](https://github.com/orgs/a-cosmology-group/discussions/254#discussioncomment-12924696)).  I suspect there is no coupling with the toroidal field.
**However**, I make the hypothesis that 'galaxy mechanical-rotation' correlates with the '***total*** magnetic field' (blue lines plus red lines), which has an influence on the intensity of the light we receive in our telescopes.

For us to observe it, light travels through the magnetic field of a distant galaxy, and then travels through the magnetic field of the Milky Way.  Light can be attenuated differently when interacting with matter, depending on the orientation of the magnetic fields it has traversed.  This would therefore correlate the observed brightness of galaxies with magnetic fields, which in turn are correlated with the physical rotation of the galaxies.

What I hope to show is that even if mechanical galaxy rotation is uncorrelated with that of the Milky Way, Shamir "sees" a correlation because of an optical effect.  That's what I am [studying with Lior](https://github.com/orgs/a-cosmology-group/discussions/254#discussioncomment-12924696) with my calculation of the total magnetic field predicted in part from stimulated transfer dynamics.

**Reply by HanDeBruijn** [reply]
@RedshiftDrift . About the first part of your response.
>When the 'mechanical' rotation direction of a galaxy is compared to the rotation of the Milky Way, the orientation or position of the observer does not matter.

![You_are_here](https://github.com/user-attachments/assets/f47dcb74-16dd-4502-be91-370d1899c1f2)

_You are here_? No Louis, I am **not** there! I am an alien on a far away exoplanet somewhere **in between** the Milky Way and the other galaxy. In such a way that the Milky Way is behind me and the other galaxy is in front of me. This is a case that you seem to have forgotten. So I stay with my point that clockwise / counter clockwise is a **subjective**, not an objective **distinction** for 'mechanical' rotations.

About the second part of your response. You (should) know that I am a proponent of your **drain** model. For two reasons: (1) drains do not cost an enormous amount of energy, (2) they might provide a mechanism explaining the hypothetical increase of mass in my [modeling](http://www.alternatievewiskunde.nl/cosmology/vervolg.htm) of flat rotation curves with galaxies.

**Reply by RedshiftDrift** [reply]
> I stay with my point that clockwise / counter clockwise is a subjective, not an objective distinction for 'mechanical' rotations.

I agree that clockwise/counterclockwise rotations are subjective, but that's not what I am talking about in this post...

**Reply by HanDeBruijn** [reply]
>I agree [ .. ] but that's not what I am talking about in this post.

Then we both agree!

**Reply by Francois-Zinserling** [reply]
@HanDeBruijn you cannot increase mass by just absorbing electrons. A massive excess of electrons means chemistry doesn't happen anymore, or at least not as we know it.

@RedshiftDrift proposes there is a wide loopback, out via the disc and back again into the 'reverse' jets , but this still puts massive amounts of electrons into space, and eventually back into the centre of the galaxy. I don't know if we have observed net (+) charged galaxy discs, or net (-) charged centres. 

I know Louis is passionate about this one but those things still look like jets to me, and still seem improbable as drains. Too long, even for twister type effects. 

Then if I follow the above posts on the toroid, and if I want to insist those are jets, then the jets must be (+) positrons, which is equally hard to comprehend. 

Sorry, this post of mine hasn't helped anyone, including myself.

**Reply by RedshiftDrift** [reply]
@Francois-Zinserling the electron current does not imply a net positive or negative charge.  A current in a metal wire does not make the wire charged.  In fact, the plasma is practically neutral everywhere as the density of protons is essentially the same as the density of electrons.  So we will not observe charged galaxies or volumes of space.

How would the positrons you postulate be created?  Anti-matter only exists for a very short time before it is annihilated by matter.  The interstellar plasma that extends to intergalactic space is mostly a neutral mix of electrons and protons.

"Looks like a jet" is likely an optical illusion.  Mixed electrons and protons have a very complex dynamics.  The light we detect from such a plasma comes from collisions between electrons and protons.  The light we detect is linked to density waves where high density of electrons and protons collide.  The motion of these density waves may not necessarily reflect the motion of the electrons nor that of the protons.

Stimulated Transfers predict the magnitude of the *outward* force that pushes electrons along the galactic disk.  These electrons can only be sucked from the IGM through the poles of the galaxy, creating an electron drain.   The magnetic field created by the current produces counter-rotating toroidal magnetic fields, circulating in the same direction as what is observed for the Milky Way.  Spectral Transfers also  produce an acceleration equal to MOND's acceleration, explaining why galaxies have constant rotation curves.

This is all very exciting.
> Louis is passionate about this one

For a good reason: "Eppure si muove!" and "Eppure gli elettroni si muovono!"
